x-pack/test/ingest_manager_api_integration/apis/fixtures/test_packages/outdated/0.1.0/dataset/test/fields/fields.yml

```diff
@@ -0,0 +1,16 @@
+- name: dataset.type
```

Comment:
Can we call this package `update_test` or something similar? Not all versions of it are outdated.

Reply:
the reason i called it "outdated" was because the only purpose of the package and versions were to test updating to an out of date package.

Reply:
It's fine, I just stumbled over it. Whatever works best for you.

Reply:
@skh sorry, i deleted my comment not realizing you responded.  i decided to rename it actually because I think I can do that error test within the update test (not yet created because of the inability to install an older package)